### ISSUE TITLE
`Pull debugsession` command does nothing when timeline not loaded

### ISSUE THREAD

**Issue by tolauwae** [TOPLLab/WARDuino-VSCode#108]
When the debug timeline view is not loaded in VS Code (it is collapsed for instance), the pull debugsession just does nothing. No error or message.

That is because of this `if` statement:

https://github.com/TOPLLab/WARDuino-VSCode/blob/be8fe97a6f112a9b69be998f4662b87c1bece33a/src/DebugSession/DebugSession.ts#L398-L400

The command should still work even if the view is hidden. Maybe the command can "launch" the view and make it visible or something similar.

**Comment by chscholl** [comment]
That's pretty bad, we should never depend on the GUI components to keep track of the time. We should have a model time which we update when the GUI changes and here we should just access the current value from our model.

**Comment by tolauwae** [comment]
@carllocos is it just a case of removing an if? Or is this more deeply embedded in the new software structure?

**Comment by carllocos** [comment]
@chscholl agree. That should indeed never depend on the GUI. I'm surprised that somehow it is the case. I made the debuggerBridge intentionally an event emitter to remove the dependency from the GUI.

> @carllocos is it just a case of removing an if? Or is this more deeply embedded in the new software structure?

I do not remember exactly. I'll have to look into the code. I will fix the issue now worries.
